app/views/layouts/application.html.slim

```diff
@@ -11,6 +11,11 @@ html
     link rel="icon" type="image/png" href=image_path("favicon.ico")
 
   body data-action=params[:action] data-controller=params[:controller]
+    javascript:
```

Comment:
My concern is this: If we put it into the normal JavaScript file it would execute before the DOM has been created. So we will use a listener on `DOMContentLoaded` (like https://github.com/brave-intl/publishers/pull/323/files#diff-b7e5bf73cff0ecca1f352a6abb55b95dR50). I'm concerned the HTML may get layout and paint before that event handler fires, and that users would see an unexpected flash of bad content before this check ran.

In most cases I agree with you that inline scripts are a bad style, however in this case I think an inline JS block is the correct approach. I can add a comment here to make clear why this exception is permitted.